test/normalization/admonition/valid-children.js

```diff
@@ -0,0 +1,56 @@
+/** @jsx h */
```

Comment:
I know that I said you can add test in `admonition` but now I think it would make more sense to have this test in `figure` - sth like `figure-valid-parents`
I'll mark @aiwenar To review since he is responsible for this repository

Reply:
I think this will be our first direct test for child-parent relationship, but I agree that it would fit better with figure tests, possibly as `figure/in-admonition.js` but I'm open to suggestions.

To be clear, this would be a test specifically for figure in admonition, and shouldn't test other elements.

Reply:
@aiwenar 
I have a feeling that making this type of tests `figure/in-{parent}` will produce a lot of test files is `figure/*` directory and it will become a little bit messy. What do you think about creating a subfolder inside `figure` which will contain only child-parent relation test?

Reply:
I don't think it's worth adding tests for every possible parent, especially since they are unrelated to this PR. We are interested in adding support for figures in admonitions, so only a test for this single case should be added.